sdk/cosmosdb/cosmos/samples-dev/ChangeFeedIterator/ChangeFeedFullFidelityEntireContainer.ts

```diff
@@ -0,0 +1,106 @@
+ * @summary Demonstrates using a ChangeFeed for entire container with AllVersionsAndDeletes mode
+ */
+
+import * as dotenv from "dotenv";
+dotenv.config();
+
+import { finish, handleError, logSampleHeader, logStep } from "../Shared/handleError";
+import {
+  CosmosClient,
+  PartitionKeyDefinitionVersion,
+  Container,
+  StatusCodes,
+  ChangeFeedStartFrom,
+  ChangeFeedMode,
+  ChangeFeedIteratorOptions,
+} from "@azure/cosmos";
+
+const key = process.env.COSMOS_KEY || "<cosmos key>";
+const endpoint = process.env.COSMOS_ENDPOINT || "<cosmos endpoint>";
+const databaseId = process.env.COSMOS_DATABASE || "<cosmos database>";
+const containerId = process.env.COSMOS_CONTAINER || "<cosmos container>";
+
+logSampleHeader("Change Feed For Entire Container With AllVersionsAndDeletes Mode");
+
+async function ingestData(container: Container, initialize: number, end: number) {
+  console.log(`beginning data ingestion from id - item${initialize} to id - item${end - 1}`);
+  for (let i = initialize; i < end; i++) {
+    await container.items.create({ id: `item${i}`, name: `sample1`, key: i });
+    await container.items.create({ id: `item${i}`, name: `sample2`, key: i });
+    await container.items.create({ id: `item${i}`, name: `sample3`, key: i });
+    await container.items.create({ id: `item${i}`, name: `sample4`, key: i });
+  }
+  console.log("ingested items");
+}
+
+async function insertAndModifyData(container: Container, initialize: number, end: number) {
+  await ingestData(container, initialize, end);
+  await container.items.upsert({ id: `item${initialize}`, name: `sample1`, key: initialize + 1 });
+  console.log(`upserted item with id - item${initialize} and partition key - sample1`);
+  await container.item(`item${initialize}`, `sample1`).delete();
+  console.log(`deleted item with id - item${initialize} and partition key - sample1`);
+}
+
+// Establish a new instance of the CosmosClient to be used throughout this demo
+const client = new CosmosClient({ endpoint, key });
+
+async function run(): Promise<void> {
+  try {
+    const { database } = await client.databases.createIfNotExists({ id: databaseId });
+    const containerDef = {
+      id: containerId,
+      partitionKey: {
+        paths: ["/name"],
+        version: PartitionKeyDefinitionVersion.V1,
+      },
+      throughput: 11000,
+    };
+    const { container } = await database.containers.createIfNotExists(containerDef);
+    let changeFeedIteratorOptions: ChangeFeedIteratorOptions;
+    changeFeedIteratorOptions = {
+      changeFeedStartFrom: ChangeFeedStartFrom.Now(),
+      changeFeedMode: ChangeFeedMode.AllVersionsAndDeletes,
+    };
+
+    let iterator = container.items.getChangeFeedIterator(changeFeedIteratorOptions);
+    logStep("Start fetching changes from now");
+    let continuationToken = "";
+    let res = await iterator.readNext();
+    console.log("Result should be empty as no changes are made yet", res.result);
+    await insertAndModifyData(container, 1, 3);
```

Comment:
add a comment stating why are we inserting and modifying data.

Reply:
addressed